tools/web3py-example/README.md

```diff
@@ -0,0 +1,53 @@
+# Web3py example
+Example scripts for basic operations
+
+### How to start
+1. **Install Solidity Compiler (`solc`)**:
+    - Install `solc` by following the instructions in the [Solidity documentation](https://docs.soliditylang.org/en/latest/installing-solidity.html).
+    - On Ubuntu, you can run:
+      ```sh
+      sudo apt install solc

```

Comment:
Is this step needed given that in `test.py` there is a `from solcx import install_solc, compile_files`?

I would assume that a function `install_solc` fetches the given compiler version.